thirdparty/DiskANN/include/diskann/aio_context_pool.h

```diff
@@ -8,14 +8,13 @@
 #include <condition_variable>
 #include "utils.h"
 #include "concurrent_queue.h"
+#include "aux_utils.h"
 
-namespace {
-  constexpr size_t default_max_nr = 65536;
-  constexpr size_t default_max_events = 256; // sync this with diskann MAX_N_SECTOR_READS
-  const size_t     default_pool_size =
-      std::min(size_t(std::thread::hardware_concurrency() * 10),
-               default_max_nr / default_max_events);
-}  // namespace
+constexpr size_t default_max_nr = 65536;
+constexpr size_t default_max_events = diskann::MAX_N_SECTOR_READS / 2;
+const size_t     default_pool_size =
+    std::min(size_t(std::thread::hardware_concurrency() * 10),
```

Comment:
To my understanding, this thread number will be host's thread number. Let's change it to a fixed number